### PR TITLE
Add information on running Prow tests

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:
- Updated `CONTRIBUTING.md` with information on running Prow tests on PRs.
- Removed references to `kyma/docs/Jenkinsfile` from the `add-new-component-docs.md` document.
- Renamed and updated the `links-in-docs.md` document to include information on how to link to files in the `assets` folders. 

**Related issue(s)**
See https://github.com/kyma-project/test-infra/issues/72 and https://github.com/kyma-project/test-infra/issues/337

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -60,19 +60,22 @@ To contribute code or content to a given Kyma repository, follow these steps:
...
+- If you are an external contributor, contact the repository maintainers specified in the `CODEOWNERS` file to review your PR and add the `/test all` comment to your PR to trigger all tests. A Kyma organization member needs to rerun the tests manually each time you commit new changes to the PR.
 
-> **NOTE:** The reviewer must check if the related CI job and tests have completed successfully before approving the pull request.
+11. Wait for the Kyma maintainers to review and approve your PR. The maintainers can approve it, request enhancements to your change, or reject it.
 
-13. When the maintainers approve your change, merge the pull request. If you are an external contributor, contact the repository maintainers specified in the `CODEOWNERS` file to do the merge.
+> **NOTE:** The reviewer must check if all related CI tests have completed successfully before approving the PR.
+
+13. When the maintainers approve your change, merge the PR. If you are an external contributor, contact the repository maintainers specified in the `CODEOWNERS` file to do the merge.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply CONTRIBUTING.md]: Really? :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Read the subsections to learn the details of the agreements to submit and licenc
...
+[Individual Contributor License Agreement](https://gist.github.com/CLAassistant/bd1ea8ec8aa0357414e8) before you contribute code or content to any Kyma repository. Kyma maintainers will not accept contributions made without such a consent. This applies to all contributors, including those contributing on behalf of a company. If you agree to the content of the Agreement, click on the link posted by the CLA assistant as a comment to the pull request (PR). Click it to review the CLA and accept it on the next screen if you agree to it. The CLA assistant saves your decision for the upcoming contributions and notifies you if there is any change to the CLA in the meantime.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply CONTRIBUTING.md]: np

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -34,18 +34,18 @@ If you are a contributor, follow these basic rules:
...
 * Work on forks of Kyma repositories.
-* Squash and rebase every pull request before merging.
-* You can merge a pull request if you receive an approval from at least one code owner from each part of the repository to which you contribute in your pull request.
+* Squash and rebase every PR before merging.
+* You can merge a PR if you receive an approval from at least one code owner from each part of the repository to which you contribute in your PR.
 
 Every contributor commits to the following agreement:
 
-* In each pull request, include a description or a reference to a detailed description of the steps that the maintainer goes through to check if a pull request works and does not break any other functionality.
+* In each PR, include a description or a reference to a detailed description of the steps that the maintainer goes through to check if a PR works and does not break any other functionality.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply CONTRIBUTING.md]: 1. sure 2. for the doc consistency reasons, I would not (wouldn't?) use the contraction here

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -34,18 +34,18 @@ If you are a contributor, follow these basic rules:
...
 * Provide clear and descriptive commit messages.
 * Follow the squash and rebase process.
 * Follow the accepted documentation rules and use appropriate templates.
-* Choose the proper directory for the content of your pull request, depending on the role of the component, and how it relates to the project structure.
-* As the creator of the pull request, you are responsible for ensuring that the pull request follows the correct review and approval flow.
-* Make sure that the person who owns the documentation, such as a technical writer, reviews the changes in the documentation before you merge the pull request.
+* Choose the proper directory for the content of your PR, depending on the role of the component, and how it relates to the project structure.
+* As the creator of the PR, you are responsible for ensuring that the PR follows the correct review and approval flow.
+* Make sure that the person who owns the documentation, such as a technical writer, reviews the changes in the documentation before you merge the PR.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply CONTRIBUTING.md]: ok

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -34,18 +34,18 @@ If you are a contributor, follow these basic rules:
...
 
-* In each pull request, include a description or a reference to a detailed description of the steps that the maintainer goes through to check if a pull request works and does not break any other functionality.
+* In each PR, include a description or a reference to a detailed description of the steps that the maintainer goes through to check if a PR works and does not break any other functionality.
 * Provide clear and descriptive commit messages.
 * Follow the squash and rebase process.
 * Follow the accepted documentation rules and use appropriate templates.
-* Choose the proper directory for the content of your pull request, depending on the role of the component, and how it relates to the project structure.
-* As the creator of the pull request, you are responsible for ensuring that the pull request follows the correct review and approval flow.
-* Make sure that the person who owns the documentation, such as a technical writer, reviews the changes in the documentation before you merge the pull request.
+* Choose the proper directory for the content of your PR, depending on the role of the component, and how it relates to the project structure.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply CONTRIBUTING.md]: I would even say that point is redundant

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -19,7 +19,7 @@ Read the subsections to learn the details of the agreements to submit and licenc
...
+[Individual Contributor License Agreement](https://gist.github.com/CLAassistant/bd1ea8ec8aa0357414e8) before you contribute code or content to any Kyma repository. Kyma maintainers will not accept contributions made without such consent. This applies to all contributors, including those contributing on behalf of a company. If you agree to the content of the Agreement, click on the link posted by the CLA assistant as a comment to the pull request (PR). The CLA assistant saves your decision for the upcoming contributions and notifies you if there is any change to the CLA in the meantime.
```
Comment (tomekpapiernik) [inline CONTRIBUTING.md]: "click on the link posted by the CLA assistant" -> I'd say "click the link" but that might be just a matter of preference.

Comment (tomekpapiernik) [inline CONTRIBUTING.md]: "CLA assistant saves your decision for the upcoming contributions" -> "for future contributions" ?

Comment (kazydek) [reply CONTRIBUTING.md]: yup

**Pull request merged**